integration/scripts/integration.sh

```diff
@@ -31,6 +31,7 @@ build_containers() {
 start_containers() {
     bash -c "${DC} down"
     docker network prune --force
+    bash -c "${DC} pull"
```

Comment:
The `up` command should implicitly perform the pull. Does it not?

Reply:
nope
if you have something with `latest` tag, up will not re-pull it